client/src/includes/sidePanel.js

```diff
@@ -62,13 +62,18 @@ export default function initSidePanel() {
           body.classList.add('side-panel-open');
         }
       } else if (!panel.hidden) {
-        // eslint-disable-next-line no-param-reassign
-        panel.hidden = true;
-        panel.dispatchEvent(new CustomEvent('hide'));
-        sidePanelWrapper.classList.remove(`form-side--${name}`);
+        const hidePanel = () => {
+          // eslint-disable-next-line no-param-reassign
+          panel.hidden = true;
+          panel.dispatchEvent(new CustomEvent('hide'));
+          sidePanelWrapper.classList.remove(`form-side--${name}`);
+        };
 
         if (panelName === '') {
           body.classList.remove('side-panel-open');
+          setTimeout(hidePanel, 500);
```

Comment:
What if the panel is reopened before this executes? I feel like we should pair this with a `clearTimeout` between lines 56 and 57 just to be on the safe side (particularly if the timeout is upped in the future), what do you think?

Reply:
@jacobtoppm good question! I’ve added a `clearTimeout` now, but I’m not sure whether this scenario is technically possible (at least currently). The panel won’t be considered to be closed until `hidePanel` is called, and attempting to open the panel multiple times in a row does nothing.